editors/kate/Jakt.xml

```diff
@@ -0,0 +1,222 @@
+                
+                <!-- Start handling single-line comment -->
+                <RegExpr attribute="Comment" context="assign" String="\/\/.*" firstNonSpace="false" />
+                
+                <!-- Start handling multiple-line string : jaktDoubleQuot -->               
+                <DetectChar attribute="String" context="jaktDoubleQuot" char="&quot;" />
+                
+                <!-- Start handling multiple-line string : jaktSingleQuot -->               
+                <DetectChar attribute="String" context="jaktSingleQuot" char="'" />
+        
+                <!-- Start handling JaktFunction -->
+                <RegExpr attribute="JaktFunction" context="assign" String="\w+[a-z-A-Z1-9](?=\s*\()" firstNonSpace="false" />
+                
+                <!-- Start handling JaktlFunctionLable -->
+                <RegExpr attribute="JaktlFunctionLable" context="assign" String="\w+(?=\s*:)(?!::)" firstNonSpace="false" />
+                
+                <!-- Start handling JaktlFunctionLable -->
+                <RegExpr attribute="JaktlSymbols" context="assign" String="(\[\[|\]\])|(\&lt;|\&gt;|-\&gt;)" firstNonSpace="false" />
+                
+                <!-- Start handling lists -->
+                <keyword attribute="JaktVisModifier" context="#stay" String="jaktVisModifier" />
+                <keyword attribute="JaktStructure" context="#stay" String="jaktStructure" />
+                <keyword attribute="JaktConstant" context="#stay" String="jaktConstant" />
+                <keyword attribute="JaktType" context="#stay" String="jaktType" />
+                <keyword attribute="JaktVarDecl" context="#stay" String="jaktVarDecl" />
+                <keyword attribute="JaktWordOperator" context="#stay" String="jaktWordOperator" />
+                <keyword attribute="JaktMacro" context="#stay" String="jaktMacro" />
+                <keyword attribute="JaktException" context="#stay" String="jaktException" />
+                <keyword attribute="JaktKeyword" context="#stay" String="jaktKeyword" />
+                <keyword attribute="JaktBoolean" context="#stay" String="jaktBoolean" />
+                <keyword attribute="JaktExecution" context="#stay" String="jaktExecution" />
+                <keyword attribute="JaktRepeat" context="#stay" String="jaktRepeat" />
+                <keyword attribute="JaktConditional" context="#stay" String="jaktConditional" />
+                
+            </context>
+            
+            <!-- End handling multiple-line string : jaktDoubleQuot -->
+            <context attribute="String" lineEndContext="#stay" name="jaktDoubleQuot">
+                <DetectChar attribute="String" context="#pop" char="&quot;" />
+            </context>
+            
+            <!-- End handling multiple-line string : jaktSingleQuot -->
+            <context attribute="String" lineEndContext="#stay" name="jaktSingleQuot">
```

Comment:
This can optionally be preceded by `c` or `b`, for c_char and u8; not sure if the syntax highlighter needs to care though.

Reply:
I will work on it.